### DISCUSSION TITLE
Should errorExchange be before or after the ssrExchange?

### DISCUSSION THREAD

**Discussion by Axedyson** [urql-graphql/urql#2366] [Q&A] [answered]
Take a look at the following code:
https://github.com/AndysonDK/syncbase/blob/main/packages/web/src/graphql/client.ts
Is the way the exchanges are setup correct? Or should the errorExchange be placed before ssrExchange. I'm unable to figure out if ssrExchange from next-urql is a synchronous or an asynchronous exchange

**Comment by JoviDeCroock** [comment, marked as answer]
I would say the errorExchange is better for after the ssr/cache as it mostly is used to intercept errors coming from fetch I'd assume
